pallets/loans/src/lib.rs

```diff
@@ -260,6 +266,11 @@ pub mod pallet {
     pub type LastAccruedInterestTime<T: Config> =
         StorageMap<_, Blake2_128Concat, AssetIdOf<T>, Timestamp, ValueQuery>;
 
+    /// Liquidation free collateral.
+    #[pallet::storage]
+    #[pallet::getter(fn liquidation_free_collaterals)]
+    pub type LiquidationFreeCollaterals<T: Config> = StorageValue<_, Vec<AssetIdOf<T>>, ValueQuery>;
```

Comment:
may not need this storage

Reply:
Why?

Reply:
1. The vault cannot be dissolved when ctoken haven't redeemed. We cannot liquidate ctoken when the vault exists.
2. The ctoken cannot be force redeemed because the owner is our pallet when ctoken is collateral.

Due to 1) and 2), we cannot use vault as equivalent of the storage.